server/package.json

```diff
@@ -12,6 +12,7 @@
   "license": "MIT",
   "dependencies": {
     "express": "^4.17.1",
+    "nodemon": "^2.0.2",
```

Comment:
this dependency "normally" is installed global in the app  `npm i -g nodemon` but if you like it in the app is in "devDependencies" with `yarn add -D nodemon`